content/en/docs/concepts/security/service-accounts.md

```diff
@@ -183,7 +183,7 @@ following methods:
   Secrets in Pods. These tokens don't expire and don't rotate.
   This method is not recommended, especially at scale, because of the risks associated
   with static, long-lived credentials. In Kubernetes v1.24 and later, the
```

Comment:
This version range can be made more specific.

Reply:
The feature gate is removed in the current versions, but not in old ones. That is how I stumbled upon the outdated link, because I was upgrading from `1.26`. So i think it is still useful to have information about it, at least for a while. I have tried to clarify the version ranges and rewrote some of the prose to be more helpful in my opinion.

I can rewrite the text further, but then it would be good to know what you would consider appropriate.